### DISCUSSION TITLE
How can I customise the notification icon as it appears in the Notifications Drawer?

### DISCUSSION THREAD

**Discussion by dg2k** [thudugala/Plugin.LocalNotification#410] [Q&A]
My notification icon, as it appears in the Notification Drawer (as you swipe down on the Status Bar), is not optimal, say, compared with other apps such as Twitter or YouTube.

- The icon size is smaller. For `IconSmallName` I tried 24x24px, 48x48px, 96x96px and even none (so the default App icon is scaled and used). No difference was noticed.

- The background setting didn't work for me. I tried setting the `Color` property of `AndroidOptions` object when creating NotificationRequest as:
`Color = new AndroidColor() { Argb = 0x0000ff }` but no success.

Any tips how to customize the icon for better visibility?

**Comment by mos379** [comment]
especially with Android 12 there are guidelines on how to design it

**Reply by dg2k** [reply]
Would you kindly provide the link for the guidelines? Thank you.

**Reply by mos379** [reply]
https://github.com/thudugala/Plugin.LocalNotification/wiki/%5BAndroid%5D-How-to-change-notification-icon
